db-tests/src/test/java/TestTriggers.java

```diff
@@ -577,4 +639,124 @@ void shouldIncrementSimulationRevisionOnSimulationUpdate() throws SQLException {
       assertEquals(initialRevision + 1, updatedRevision);
     }
   }
+
+  @Nested
+  class PlanDatasetTriggers {
+    @Test
+    void shouldCreateDefaultDatasetOnPlanDatasetInsertWithNullDatasetId() throws SQLException {
+      final var res = connection.createStatement()
+          .executeQuery(
+              """
+                  INSERT INTO plan_dataset (plan_id, offset_from_plan_start)
+                  VALUES (%s, '0')
+                  RETURNING dataset_id;"""
+                  .formatted(planId)
+          );
+      res.next();
+      final var newDatasetId = res.getInt("dataset_id");
+      res.close();
+
+      assertInstanceOf(Integer.class, newDatasetId);
```

Comment:
🔶 I'm interested to discuss this assertion choice. I may be missing something here but it seems like this is always true given the return type of the `res.getInt` call. Could it be more descriptive to test something about the value like maybe check `newDatasetId > 0` or whatever we expect it to be?

Reply:
Done